css/style.css

```diff
@@ -1,84 +1,227 @@
+Project Name: Home Page
+Client: Relaxr
+Author: Jessica Lewis
+Developer @GA in DC
 */
 
-
+/* CSS Reset 
+#element { 
+  margin:0; 
+  padding:0; 
+  font-size:100%; 
+  line-height:1; 
+}
+...
+#element rules: 
+#element { 
+  margin:5px 0 10px; 
+  font-size:13px; 
+  line-height:1.5; 
+}
+*/
 /******************************************
 /* SETUP
 /*******************************************/
 
 /* Box Model Hack */
 * {
-     -moz-box-sizing: border-box; /* Firexfox */
-     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
-     box-sizing: border-box; /* IE */
+ -moz-box-sizing: border-box; /* Firexfox */
+ -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+ box-sizing: border-box; /* IE */
 }
 
-/* Clear fix hack */
+/* Clear fix hack
 .clearfix:after {
-     content: ".";
-     display: block;
-     clear: both;
-     visibility: hidden;
-     line-height: 0;
-     height: 0;
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
 }
-
 .clear {
   clear: both;
 }
-
 .alignright {
   float: right;
-  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+  padding: 0 0 10px 10px; note the padding around a right floated image
 }
-
 .alignleft {
   float: left;
-  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+  padding: 0 10px 10px 0; note the padding around a left floated image 
 }
-
+*/
 /******************************************
 /* BASE STYLES
 /*******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+ color: #000;
+ font-size: 12px;
+ line-height: 1.4;
+ font-family: 'Open Sans', sans-serif; 
+}
+button {
+  border-radius: 5px;
+  padding: 8px 35px;
+  text-align: center;
+  margin: 10px;
 }
-
-
 /******************************************
 /* LAYOUT
 /*******************************************/
 
-/* Center the container */
-#container {
-  width: 100%;
-  margin: auto;
+/* Center the Container */
+#container {  
   height: 500px;
-
+  width: 900px;
+  margin: auto;
 }
-
 header {
+  background: url(../images/header_bg.jpg) no-repeat center top fixed rgb(134,141,157);
+  min-height: 500px;
+}
+.header-container {
+  width: 800px;
+  margin: auto;
+  text-align: center;
+  padding: 40px 20px;
+}
+.benefits {
+  background: #f0efef;
+}
+.benefits-container {
+  width: 800px;
+  margin: auto;
+  padding: 40px 40px;
+}
+.testimonial {
+  background: url(../images/testimonial_bg.jpg);
+}
+.testimonial-container {
+  width: 40%;
+  margin: auto;
+  text-align: center;
+  padding: 40px 30px;
+}
+.sign-up {
+  background: #f9e42e;
+  text-align: center;
+}
+.sign-up-containter {
+  width: 40%;
+  margin: auto;
+  padding: 40px 30px;
+}
+footer {
+  background: #000;
```

Comment:
Thanks for letting me know about this one too. Also committed changes to my pull request. Hopefully I did it right! 

- Jessica Lewis

> On Feb 27, 2016, at 4:26 PM, ericmasiello <notifications@github.com> wrote:
> 
> In css/style.css:
> 
> > +  width: 40%;
> > +  margin: auto;
> > +  text-align: center;
> > +  padding: 40px 30px;
> > +}
> > +.sign-up {
> > +  background: #f9e42e;
> > +  text-align: center;
> > +}
> > +.sign-up-containter {
> > +  width: 40%;
> > +  margin: auto;
> > +  padding: 40px 30px;
> > +}
> > +footer {
> > +  background: #000;
> I think this is supposed to be #121212 for the background-color
> 
> —
> Reply to this email directly or view it on GitHub.
>